app/src/main/res/values-de/strings.xml

```diff
@@ -226,6 +229,10 @@
     <string name="mark_notification_summary">Benachrichtigung als gelesen markieren sobald sie angetippt wurde.</string>
     <string name="theme_title">Wähle ein Design</string>
     <string name="theme_summary">Wähle das Standarddesign aus</string>
+	<string name="theme_color_summary">Wähle eine Akzentfarbe</string>
+    <string name="theme_color_title">Akzentfarbe</string>
+    <string name="logo_designer" formatted="true" translatable="false">Logo Designer (%s)</string>
+    <string name="google_plus" translatable="false">Google+</string>
```

Comment:
Hmm I dont know
Could be I am wrong but what I understood for @k0shk0sh they are supposed to be removed.

I will merge it and if its an issue he will let you know.

Thanks for your input